### ISSUE TITLE
rtl8723be: Wifi repeatedly connect-disconnect on Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by afwanwh** [lwfinger/rtlwifi_new#159]
Hi,

I have installed from this repository on my laptop (using Ubuntu 16.04). The installation was success. After installation and reboot, I tried use my wifi but the connection repeatedly connect-disconnect.

So, I added this to my system.
`echo "options rtl8723be fwlps=0" | sudo tee /etc/modprobe.d/rtl8723be.conf`
But, it didn't work too.

Any suggestion?

**Comment by lwfinger** [comment]
Yes, change that fwlps to ips. Most people have more success disabling power save completely. What you did was change from firmware control to software control.

**Comment by afwanwh** [comment]
Didn't work too even I change ips to 0.

**Comment by lwfinger** [comment]
Have you checked your signal strength? If not, then run the command

sudo iw dev wlan0 scan | grep signal

Change "wlan0" to match the name for your wireless device. Please report the output.

**Comment by afwanwh** [comment]
My wifi signal strength was very good. But I will test with other wifi. Currently I have bad weather and it affects to internet connection. I'll report later.

**Comment by lwfinger** [comment]
OK. As you do not provide the numbers, I will not worry about your problem any more. I spend a lot of time taking care of complaints such as yours, but I have no patience for people that do not do what I ask!!

**Comment by afwanwh** [comment]
It's OK now, no problem after change to ips=0. (I tested on another wifi)

First, I would say thank to you.
Second, I would like to say sorry for not providing any numbers. It was connection issue due to bad weather when I tested.
Third, you don't have to worry about this issue. I was not complaining, I just ask suggestion to you. Even if you didn't reply this issue, it's OK for me.

You are not paid for help me, so I didn't complain but ask suggest (in my first message, I say "Any suggestion?").
